azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/localuser/DragTouchListener.kt

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.presentation.fragment.calling.localuser
+
+import android.view.MotionEvent
+import android.view.View
+
+class DragTouchListener : View.OnTouchListener {
+
+    private var curX = 0f
+    private var curY = 0f
+    private var startX = 0f
+    private var startY = 0f
+    private var tranX = 0f
+    private var tranY = 0f
+    private lateinit var view: View
+
+    internal constructor() {}
```

Comment:
can we convert this to primary constructor

Reply:
addressed now.